src/signalo/edge_cases/models.py

```diff
@@ -46,4 +47,20 @@ class DifferentSrid(models.Model):
     """This model uses a different SRID from the others, allowing to check that this is properly managed by QGIS."""
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    geom = models.PointField(srid=2056)
+    geom = models.PointField(srid=2154)
+
+
+@register_oapif_viewset(
+    custom_serializer_attrs={"permission_classes": (permissions.AllowAny,)}
+)
+class AllowAnyModel(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    geom = models.PointField(srid=2154)
+
+
+@register_oapif_viewset(
+    custom_serializer_attrs={"permission_classes": (permissions.IsAdminUser,)}
+)
+class IsAdminUserModel(models.Model):
```

Comment:
Same (Add a docstring to document what this is (that docstring is exposed as the layer's description))